WorkflowUI/Sources/ViewControllerDescription/ViewControllerDescription.swift

```diff
@@ -18,8 +18,12 @@
 
     import UIKit
 
-    /// A ViewControllerDescription acts as a recipe for building and updating a
-    /// specific UIViewController.
+    /// A ViewControllerDescription acts as a recipe for building and updating a specific UIViewController.
+    ///
+    /// You usually use a `ViewControllerDescription` in conjunction with a `DescribedViewController`,
+    /// which will internally create and manage the described view controller for its current view controller description.
+    /// However, you can also directly invoke the public methods such as `buildViewController()`,
+    /// `update(viewController:)`, if you are manually managing your own view controller hierarchy.
```

Comment:
I'd probably be good to first describe what I'd imagine is the most frequently use of this interface: providing a description of the backing view in a `Screen`. I typically see `Screen`s, not `ViewControllerDescription`s, passed to `DescribedViewController`.

Reply:
Sounds good! I can drop in some code samples

Reply:
Took a shot at updating the docs of use cases before adding code instead.